### PR TITLE
DS-280 Implement 2nd batch of icons

### PR DESCRIPTION
**Pull request by david-cho-lerat-sonarsource**

### CODE REVIEW

File: src/components/icons/IconStar.tsx [hunk 1]
```diff
@@ -22,9 +22,5 @@ import { IconMaterialWrapper, IconProps } from './IconWrapper';
 
 export function IconStar(props: Readonly<IconProps>) {
   // This is Material Symbols' "kid_star" icon
-  return (
```
Comment (gregaubert) [inline src/components/icons/IconStar.tsx:25]: There are actually two versions of star

**Review by gregaubert — APPROVED** [review]
Nice!

File: src/components/icons/IconStar.tsx (hunk 1 above)
Comment (david-cho-lerat-sonarsource) [reply src/components/icons/IconStar.tsx:25]: now handled with the export of `isFilled`, same for `IconHome`

**Review by gregaubert — APPROVED** [review]
Nice update!

**Pull request merged**